### ISSUE TITLE
Unreachable panic starting RGB daemon

### ISSUE THREAD

**Issue by claudiosdc** [LNP-BP/rgb-node#125]
When starting version 0.2 of the RGB daemon (`rgdb` binary), the program panics with the following message:

```shell
thread 'main' panicked at 'internal error: entered unreachable code', src/rgbd/runtime.rs:210:9
```

**Comment by claudiosdc** [comment]
After further investigation, I realized that this issue is taking place because the main thread is waiting on futures that were supposed to resolve only after the child processes (`stashd` and `fungibled`) terminate, but instead are being immediately resolved, and this is due to the fact that, at some point, the `tokio::process` module that was used to start those child processes has been replaced with the `std::process` module.

A proposed fix (#126) reverts that change.

**Comment by dr-orlovsky** [comment]
thank you for the investigation; pls see my commend in the PR
